tldw_Server_API/app/core/RAG/rag_service/user_personalization_store.py

```
@@ -97,7 +153,7 @@ def get_prior(self, doc_id: str) -> float:
         except Exception:
             return 0.0
 
-    def boost_documents(self, documents: List[any], *, corpus: Optional[str] = None) -> List[any]:
+    def boost_documents(self, documents: List[Any], *, corpus: Optional[str] = None) -> List[Any]:
```

Ruff (0.14.10)

Unused method argument: `corpus`

(ARG002)